tests/input/input_diffuse.cpp

```diff
@@ -49,9 +49,8 @@ TEST_CASE("Input reading using GetKw for an input file for a diffuse environment
   Input parsedInput = Input(filename);
   std::string units = "ANGSTROM";
   int CODATAyear = 1998;
-  std::string type = "WAVELET";
-  int patchLevel = 1;
-  double coarsity = 0.3;
+  std::string type = "GEPOL";
+  double area = 0.3 * angstromToBohr() * angstromToBohr();
```

Comment:
There is an `angstrom2ToBohr2()` function available for this in `PhysicalConstants.cpp`